packages/@ama-sdk/generator-sdk/src/generators/shell/templates/base/package.json

```diff
@@ -123,5 +123,5 @@
     "npm": ">=6.0.0",
     "yarn": ">=3.1.0"
   },
-  "packageManager": "yarn@3.5.1"
+  "packageManager": "yarn@3.6.0"
```

Comment:
should we talk to Renovate about this ?

Reply:
Supposed to be the case :S
<img width="361" alt="image" src="https://github.com/AmadeusITGroup/otter/assets/2467012/da8b9916-8639-4253-bbbe-c1aa0e7c125e">